app/src/main/java/com/aws/amazonlocation/AmazonLocationApp.kt

```diff
@@ -25,9 +28,8 @@ class AmazonLocationApp : Application() {
     }
     private fun setupUncaughtExceptionHandler() {
         Thread.setDefaultUncaughtExceptionHandler { _, throwable ->
+            mPreferenceManager.setValue(KEY_APP_CRASH, true)
             notifyAppCrash(throwable.message)
```

Comment:
Is it recording stack trace as well. I think we should send stack trace as well

Reply:
@zeeshanmakeen As we don't show stack traces to users, we don't need the full stack trace. Instead, we will display a common message after the restart.